.gitignore

```diff
@@ -90,4 +90,7 @@ bndata
 .vscode
 
 # Python virtual environment
-.venv
\ No newline at end of file
+.venv
+
+# Test data
+!tests/test_flow/test_persistence_compatability/artifacts/*/*/*.pkl
```

Comment:
I think you can also use `artifacts/**/.pkl` if you want to be less dependent on the cache's file hierarchy.

Reply:
Oh, also, nit: s/compatability/compatibility/.